<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Master Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_master_password">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_password">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter new master password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repeat_password">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Repeat new master password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;üîê Master Password Notice&lt;/span&gt;&lt;br/&gt;Your master password is the only way to access your data.&lt;br/&gt;All information is encrypted locally and &lt;span style=&quot; font-weight:700;&quot;&gt;cannot be &lt;br/&gt;recovered&lt;/span&gt; if the password is lost.&lt;br/&gt;&lt;br/&gt;‚ö†Ô∏è Please store your master password in a secure place.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toggle_password_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Passwords</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
